components/wifi/esp8266-driver/ESP8266/ESP8266.cpp

```diff
@@ -71,6 +74,7 @@ ESP8266::ESP8266(PinName tx, PinName rx, bool debug, PinName rts, PinName cts)
     _parser.oob("UNLINK", callback(this, &ESP8266::_oob_socket_close_err));
     _parser.oob("ALREADY CONNECTED", callback(this, &ESP8266::_oob_conn_already));
     _parser.oob("ERROR", callback(this, &ESP8266::_oob_err));
+    _parser.oob("ready", callback(this, &ESP8266::_oob_reset));
```

Comment:
I have doubts about the naming here. I understand that currently we only care about "ready" message from ESP8266 in the context of resetting, but the [AT documentation](https://www.espressif.com/sites/default/files/documentation/4a-esp8266_at_instruction_set_en.pdf) claims that `ready` just means "The AT firmware is ready."
How about, just in case, we call the handler `_oob_ready`, so that 
1) it reflects the purpose better (handling "ready command", just like `_oob_conn_already` is handling "ALREADY CONNECTED")
2) It can be used in the future in case it turns out ESP8266 prints ready when it's ready to do something else than reset :)

Reply:
ok, I'll change the name.